estate/models/estate_property.py

```diff
@@ -0,0 +1,103 @@
+from odoo import api, models, fields
+from odoo.exceptions import UserError
+import datetime
+from dateutil.relativedelta import relativedelta
+from odoo.tools.float_utils import float_compare
```

Comment:
Same here.

You must also place built-in imports before external imports and external imports before internal imports. A built-in import is when you import code that comes with Python by default (such as datetime). An external import occurs when you import code from outside your project (such as dateutil). And an internal import occurs when you import a code that is present in your project, but in another file.

```suggestion
import datetime

from dateutil.relativedelta import relativedelta

from odoo import api, fields, models
from odoo.exceptions import UserError
from odoo.tools.float_utils import float_compare
```